src/components/select/select-screen.js

```diff
@@ -86,6 +86,16 @@ export class Select extends Screen {
         return stateDefinition === undefined || stateDefinition.enabled !== false;
     }
 
+    addOverlay(key) {
+        this.singleItemMode.shutdown();
+        super.addOverlay(key);
+    }
+
+    _onOverlayRemoved(overlay) {
+        super._onOverlayRemoved(overlay);
+        this.singleItemMode = singleItemMode.create(this);
+    }
```

Comment:
Shouldn't need this stuff if we use makeAccessible - I'll have a look at a way to do this today